Feature/about us

grytsayo opened the pull request:
Just for test

grytsayo committed:
About-mobile w/image

grytsayo committed:
Start about section

Start coding About section

ohrimovich commented on src/partials/about-us.html:
```diff
@@ -0,0 +1,30 @@
+<section class="about">
+    <div class="container">
+        <!-- <picture>
```
Не вірний шлях до картинок ось так srset=''images/desktop/about-us_desktop.png'' і всі інші по аналогії

grytsayo replied:
srset=''images/desktop/about-us_desktop.png''   малось на увазі  srcset=?

ohrimovich replied:
Так, я там помилився

ohrimovich commented on src/partials/about-us.html:
```diff
@@ -0,0 +1,30 @@
+            Oh My Donut is the shop which adores tasty deserts and spend hours 
+            to create sophisticated donuts which will save the world!
+        </p>
+        <div style="height:25px"></div>
```
Не вірно реалізований відступ
Краще задати about__title:not(:last-child)margin-bottom: 25px; в css
Це позбавить нас потреби задавати відступ для абзацу, після натискання кнопки read more

grytsayo replied:
:not(:last-child)margin-bottom: 25px не працює. Я власне із-за того укав інший спосіб

ohrimovich replied:
Повинно працювати, скинь в комент код ситлів

ohrimovich replied:
Можеш додати p в ul і тоді, все запрацює

ohrimovich commented on src/partials/about-us.html:
```diff
@@ -0,0 +1,30 @@
+            Eva and our shop have received multiple awards & recognition for excellence. 
+        </p>
+
+        <button class="btn-more">
```
У кнопки відсутній атрибут type="button"

ohrimovich commented on src/sass/_about-us.scss:
```diff
@@ -0,0 +1,30 @@
+.about__title {
+    font-family: Caveat Brush;
+    font-weight: normal;
```
font-weight задається числом
font-weight: 400;

ohrimovich commented on src/sass/_about-us.scss:
```diff
@@ -0,0 +1,30 @@
+
+.about__text {
+    font-family: Montserrat;
+    font-weight: normal;
```
В даному випадку font-weight не потрібно задавати, ці стилі наслідуються з body

ohrimovich commented on src/sass/_about-us.scss:
```diff
@@ -0,0 +1,30 @@
+.btn-more {
+    font-family: Caveat Brush;
+    font-style: normal;
+    font-weight: normal;
```
font-weight задається числом
font-weight: 400;

grytsayo committed:
bagfix images source/font-weight/btn type

grytsayo committed:
bagfix mb in mobile + tablet

Bagfix interline space in article and tablet layout

grytsayo committed:
Bagfix done

<p> tag added in the list

grytsayo committed:
Bugfix

article changed to div
div removed from picture
id moved do section
button in BEM
